### ISSUE TITLE
Rog Ally regular gamepad stop working after installing HC

### ISSUE THREAD

**Issue by marceloadsj** [Valkirie/HandheldCompanion#658]
Also reported here: https://www.reddit.com/r/ROGAlly/comments/14i4tks/keys_doesnt_work/

It seems that after installing HC, the regular gamepad from Rog Ally stop working, even when the HC is not running at all (all configs disabled, no services running, no HC panel running as well, etc).

One person mentioned on Reddit above to restart an Armoury Crate Service, but doesn't work for me. Also restarting the machine doesn't work (even with HC without auto-start).

The only way I can just use the regular gamepad after installing HC is with HC running and enabled.

Is there a way to just turn it off completely, and going back to the device's gamepad, without having to uninstall HC, please?

PS: in any case thx for your work on it. The gyro is incredible 😁

**Comment by CasperH2O** [comment]
Hey @marceloadsj , it reads like something went wrong with unhiding the physical controller. HC does this through the usage of HidHide. Perhaps you disabled this on the HC Controller page, there is a toggle for it. 

If you want to do this manually, be sure HC is not running and open the application called HidHideConfigurationClient, go to the second tab and unlock the controller there. Note this this should only be done under extreme circumstances.

**Comment by Valkirie** [comment]
> Hey @marceloadsj , it reads like something went wrong with unhiding the physical controller. HC does this through the usage of HidHide. Perhaps you disabled this on the HC Controller page, there is a toggle for it.
> 
> If you want to do this manually, be sure HC is not running and open the application called HidHideConfigurationClient, go to the second tab and unlock the controller there. Note this this should only be done under extreme circumstances.
> 
> Thank you, the gyro has been quite a lot of effort over time. We plan to make it even better with gyro as a mouse.

Indeed, loosing full access to controller usually means the virtual controller is not doing its job (we're investigating) and physical controller is hidden (check status on home page). We've got a few fixes on the way to greatly improve the experience here.

**Comment by marceloadsj** [comment]
Awesoooome, you rock! Haha

I managed to "disable" HC and just use the regular gamepad via that HidHide. For now I will keep enabling/disabling HC for when I need gyro (I'm playing Doom Eternal hehe), and when I want to test other features. :)

I don't know why the HC didn't unhide the controller when it is terminated tho. But since now I have a way to toggle it, for me, you can close this issue if needed.

I'm looking forward to get new enhancements. :) Thx a lot for the support to the community.

**Comment by CasperH2O** [comment]
Issue resolved, closing.
